### DISCUSSION TITLE
Can't find crazyradio dongle in bitcraze vm

### DISCUSSION THREAD

**Discussion by cc299792458** [bitcraze/discussions#201] [Q&A]
Hi! I'm a new to bitcraze vm and I've tried to follow the tutorial "getting-started-with-development". However, when I try to flash the firmware on my crazyflie, it says "Failed to flash: Cannot find a Crazyradio Dongle". 
I don't know what's the right way to set usb in vm. I have tried that in both virtual box and VMware but failed.
Any help would be appreciate! Thanks in advance!

**Comment by knmcguire** [comment]
Hi!

Can't you see the Crazyradio PA in the usb dropdown menu? Also you need to install the extention package of virtual box to be able to use the USB device.

![Screenshot 2022-05-25 085449 (1)](https://user-images.githubusercontent.com/11061260/184848555-9936a33e-0f0e-43fc-99a2-cfc2008ae34d.png)

Don't forget to also take look at the old forum: https://forum.bitcraze.io/search.php?keywords=virtual+box+crazyradio

Also the readme has some tips for optimizing the VM. https://github.com/bitcraze/bitcraze-vm

**Reply by cc299792458** [reply]
Hi!
Thanks for your help! I finally find out that it's the problem with my virtual machine's usb setting.
I use VMware instead and connect the usb in the right down side of it.
![image](https://user-images.githubusercontent.com/52767591/185051206-1558ea38-cf84-457f-b675-b67d9a370237.png)

Really appreciate! Thanks again for your answer!

**Reply by knmcguire** [reply]
ah great. Did you had to do a lot to optimize the use of VMware? Lot of people are experiencing problems with virtualbox so maybe we should start advising VMware instead

**Reply by cc299792458** [reply]
No need to do a lot of modification. In my case, I think they are both easy to use except setting usb. VMware is a little more convenient for me :).

**Reply by knmcguire** [reply]
alright good to know! thanks.

Good luck with your work and happy hacking!

**Reply by cc299792458** [reply]
Thanks!
